### PR TITLE
Feature/1748 geo backgrounds

### PR DESCRIPTION
**Pull request by joonasnuutinen**
Implements #1748.

https://user-images.githubusercontent.com/18427577/218730764-20eca7a3-36c9-4566-8902-3f0ae0779cad.mov

Note:
- if Landsat is selected, choosing year 2012 or older causes an error
- if only Sentinel is selected, choosing a year 2016 or older causes an error

The error is:
```
Pattern 'red' did not match any bands.
```

In addition, Sentinel is still quite heavy and may lead to errors if the map is zoomed out far enough.

However, these errors are related to Sepal and not the actual UI components, so maybe these should be addressed in another ticket.

### CODE REVIEW

**Comment by joonasnuutinen** [comment]
Also, Finland is still hardcoded. Let's fix this when #2205 is merged.

File: src/client/pages/Geo/GeoMap/GeoMapMenuMosaic/geoMapMenuMosaic.tsx [hunk 1]
```diff
@@ -1,63 +1,59 @@
...
   const mosaicUrl = useMosaicUrl()
   const mosaicOptions = useMosaicOptions()
+  const mosaicSelected = useMosaicSelected()
   const map = useGeoMap()
+  const mapControllerRef = useRef<MapController>(new MapController(map))
+  const mosaicLayerKey = 'mosaic'
+
+  // Mosaic layer toggled, mosaicUrl updated or mosaicOptions changed
+  useEffect(() => {
+    // console.log('mosaic', mosaicSelected)
```
Comment (yaguzmang) [inline src/client/pages/Geo/GeoMap/GeoMapMenuMosaic/geoMapMenuMosaic.tsx]: Should we remove the console.logs?

**Review by yaguzmang — DISMISSED** [review]

**Comment by yaguzmang** [comment]
Tested. Works as expected, good job! 👍

**Comment by joonasnuutinen** [comment]
Conflicts resolved and mosaic country set dynamically.


https://user-images.githubusercontent.com/18427577/219018940-31a58cc2-49d6-432c-af4f-de02ae2394b8.mov

**Review by yaguzmang — APPROVED** [review]

**Review by minotogna — APPROVED** [review]

**Pull request merged**